server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/UserRoutesTest.java

```diff
@@ -263,6 +263,19 @@ void putShouldReturnBadRequestWhenEmptyUserName() {
                 .statusCode(HttpStatus.NOT_FOUND_404);
         }
 
+        @Test
+        void putShouldReturnBadRequestWhenUsernameDoesnHaveDomainPart() {
```

Comment:
Only true when virtualHosting is enabled.

We may need a similar test with a disabled virtualHosting.